build/optaplanner-build-parent/pom.xml

```diff
@@ -178,6 +180,21 @@
         <artifactId>jfreechart</artifactId>
         <version>${version.org.jfree.jfreechart}</version>
       </dependency>
+      <dependency>
+        <groupId>org.apache.derby</groupId>
+        <artifactId>derby</artifactId>
```

Comment:
Since we're using H2 everywhere and it seems to work fine with OpenJPA (at least in Drools), if possible I would give a try to use H2 also here and avoid adding another dependency.